agent/tick.go

```diff
@@ -75,6 +77,7 @@ func (t *AlignedTicker) next(now time.Time) time.Duration {
 		d = t.interval
 	}
 	d += internal.RandomDuration(t.jitter)
+	d += t.offset
```

Comment:
To me, it looks more logical to switch these. In my mind i'm doing offset+jitter instead of jitter+offset.